packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts

```diff
@@ -44,11 +44,15 @@ export class GraphbackCoreMetadata {
 
   public addResolvers(resolvers: IResolvers) {
     if (resolvers) {
-      this.resolvers.push(resolvers);
+      const mergedResolvers = [
+        this.resolvers,
+        resolvers
+      ]
+      this.resolvers = mergeResolvers(mergedResolvers)
```

Comment:
A kinda fancy version of Object.assign ;P

Reply:
Object.assign does not merge deep, neither does the spread operator. So you would have to do:

```ts
this.resolvers = {
  Query: {
    ...this.resolvers.Query,
    ...resolvers.Query
  },
  Mutation: {
    ...this.resolvers.Mutation,
    ...resolvers.Mutation
  }
}
```

And it would get trickier trying to merge custom root resolvers.